test/TestUtilities/Test.Utility/Commands/ProjectTestHelpers.cs

```diff
@@ -302,6 +302,25 @@ public static PackageSpec GetPackageSpec(string projectName, string rootPath = @
             return packageSpec;
         }
 
+        public static PackageSpec GetPackageSpec(string projectName, string rootPath, string framework, string dependencyName)
+        {
+            const string referenceSpec = @"
+                {
+                    ""frameworks"": {
+                        ""TARGET_FRAMEWORK"": {
+                            ""dependencies"": {
+                                ""DEPENDENCY_NAME"" : ""1.0.0""
+                            }
+                        }
+                    }
+                }";
+
+            var spec = referenceSpec.Replace("TARGET_FRAMEWORK", framework).Replace("DEPENDENCY_NAME", dependencyName);
```

Comment:
I'm curious why replacing is better than string interpolation. Are you following current code pattern in other tests?

```suggestion
            var spec = $@"
                {{
                    ""frameworks"": {{
                        ""{framework}"": {{
                            ""dependencies"": {{
                                ""{dependencyName}"" : ""1.0.0""
                            }}
                        }}
                    }}
                }}";
```